tests/integration/lxd/test_lxd.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Canonical Ltd.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License version 3 as
+# published by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+
+import pytest
+
+from craft_providers.lxd import LXD
+
+
+@pytest.fixture()
+def lxd():
+    yield LXD()
+
+
+def test_init(lxd):
+    lxd.init(auto=True)
+    lxd.init(sudo=True, auto=True)
```

Comment:
no assertion to verify it is actually initialized?

Reply:
this is really just checking that init completed without error.  what would you recommend to verify? something like an lxc list?

when I get to the installer tests, there will be a more in-depth test where we install/configure lxd from scratch.